packages/dashmate/src/listr/tasks/setup/setupRegularPresetTaskFactory.js

```diff
@@ -44,7 +44,13 @@ function setupRegularPresetTaskFactory(
     return new Listr([
       {
         task: (ctx) => {
-          ctx.config = configFile.getConfig(ctx.preset);
+          const config = configFile.getConfig(ctx.preset);
```

Comment:
I think we don't have all testnet nodes to have moniker testnet )). Let's keep it null